docs/DeveloperGuide.adoc

```diff
@@ -225,7 +226,7 @@ This section describes some noteworthy details on how certain features are imple
 
 [[StudentView]]
 === Student View
-*Student view* is the term used to characterise the different functionalities
+*Student View* is the term used to characterise the different functionalities
```

Comment:
The Student View displays all the students, groups, and modules within TA-Tracker?
Displays all the actions related to students in the TA-Tracker?

Its a bit wordy...